beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/README.md

```diff
@@ -1,2 +1,119 @@
+
+| Input name | Description | Type | 
+| --- | --- | --- |
+| input_cram (optional)| A single mapped CRAM file; alternatively input can be a mapped BAM (`input_bam`) or unmapped BAM (uBAM; `ubam`). If using a CRAM file aligned to a reference different than the reference chosen for the GDC workflow (the `ref_fasta` input), than you must specify both the `cram_ref_fasta` and the `cram_ref_fasta_index` inputs | File |
+| input_bam (optional)| A single mapped BAM file; alternatively input can be a CRAM (`input_cram`) or uBAM (`ubam`) | File |
+| cram_ref_fasta (optional)| The reference file that was used to align the CRAM input (if used); if unspecified, the workflow will use the `ref_fasta` input file by default | File |
+| cram_ref_fasta_index (optional) | CRAM reference FASTA index for the `cram_ref_fata` if CRAM is used as workflow input | File |
+| output_map (optional)| Tab-separated file containing two columns: a list of all the read group IDs found in the input_cram (or input_bam) and a list of the desired name of the uBAMs generated for read group | File |
+| unmapped_bam_suffix (optional) | Optional string used to name the output uBAM file | String |
+| ubam (optional)| A single uBAM file; alternatively, input can be a mapped BAM (`input_bam`) or CRAM (`input_cram`) | File |
+| contamination_vcf | VCF file of common variant sites that is used for the check_contamination task | File |
+| contamination_vcf_index | Index file for the `contamination_vcf` input | File |
+| dbsnp_vcf | VCF file of known variation sites that can be used to exclude these sites from the analysis; used for the gatk_baserecalibrator task | File |
+| dbsnp_vcf_index | Index file for the `dbsnp_vcf` input | File |
+| ref_fasta | Reference FASTA used to convert an input CRAM to BAM; if CRAM is not used, this file does not need to be specified | File | 
+| ref_fai | Reference FASTA index for the `ref_fasta` input | File |
+| ref_dict | BWA reference dictionary used for alignment| File |
+| ref_amb | BWA reference file used for alignment | File |
+| ref_ann | BWA reference file used for alignment | File |
+| ref_bwt | BWA reference file used for alignment | File |
+| ref_pac | BWA reference file used for alignment | File |
+| ref_sa | BWA reference file used for alignment | File |
+
+### GDC Reference Genome
+The GDC uses the human reference genome GRCh38.d1.vd1 for all data processing. Unlike the GRCh38 reference used by WARP pipelines for production, the GDC reference includes decoy viral sequences for ten types of human viral genomes. You can learn more about the reference from the [GDC documentation](https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/#reference-genome). 
+
+The reference files required for the GDC workflow are hosted in a [GDC public Google Bucket](https://console.cloud.google.com/storage/browser/gcp-public-data--broad-references/hg38/v0/gdc). 
```

Comment:
I'm probably being nit-picky but it's not actually the GDC's bucket, it's google's public data bucket that holds all of the Broad references files. The GDC probably hosts the files elsewhere, too.

Reply:
Yes, the text was misleading- updated to remove GDC.